bin/agent_runner.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+# This process is used to maintain Huginn's upkeep behavior, automatically running scheduled Agents and
+# periodically propagating and expiring Events. It also running TwitterStreamAgents and Agents that support long running
+# background jobs.
+
+Dotenv.load if Rails.env == 'development'
```

Comment:
Foreman, right! I was wondering why I did not have issues like this before. Do you think it's worth keeping? In development I like to start the background worker without foreman to be able to restart them faster.